client/packages/programs/src/JsonForms/components/Search/SearchWithUserSource.tsx

```diff
@@ -53,10 +53,10 @@ export const SearchWithUserSource = (
   const getOptionLabel = (data: PatientSchema) =>
     options?.optionString
       ? formatTemplateString(options?.optionString, data)
-      : `${data['code']} - ${data['firstName']} ${data['lastName']}`;
+      : `${data['code'] ?? ''} - ${data['firstName']} ${data['lastName']}`;
```

Comment:
Code wasn't required.... so it showed up as undefined for me 👀  probably a better way to display this undefined value.... or make code required?

Reply:
If there is no code we shouldn't show the extra " - ", right? e.g.

```
`${data['code'] ?? `${data['code']} - `}${data['firstName']} ${data['lastName']}`;
```

Reply:
Good idea~